neuralforecast/losses/pytorch.py

```diff
@@ -560,32 +575,41 @@ def __call__(
         **Returns:**<br>
         `mqloss`: tensor (single value).
         """
-        if mask is None:
-            mask = torch.ones_like(y_hat)
-
-        n_q = len(self.quantiles)
 
         error = y_hat - y.unsqueeze(-1)
         sq = torch.maximum(-error, torch.zeros_like(error))
         s1_q = torch.maximum(error, torch.zeros_like(error))
-        mqloss = self.quantiles * sq + (1 - self.quantiles) * s1_q
+        losses = (1 / len(self.quantiles)) * (
+            self.quantiles * sq + (1 - self.quantiles) * s1_q
+        )
 
-        # Match y/weights dimensions and compute weighted average
-        mask = mask / torch.sum(mask)
-        mask = mask.unsqueeze(-1)
-        mqloss = (1 / n_q) * mqloss * mask
-        return torch.sum(mqloss)
+        losses = losses.swapaxes(
```

Comment:
The swapaxes solution seems to overlap with the previous weighted_mean used in DistributionLoss